src/CMakeLists.txt

```diff
@@ -100,16 +100,11 @@ if(UMF_POOL_JEMALLOC_ENABLED)
                                        "UMF_POOL_JEMALLOC_ENABLED=1")
 endif()
 
-if(NOT UMF_DISABLE_HWLOC)
-    set(UMF_SOURCES ${UMF_SOURCES} ${HWLOC_DEPENDENT_SOURCES}
-                    memtargets/memtarget_numa.c)
-    set(UMF_LIBS ${UMF_LIBS} $<BUILD_INTERFACE:${UMF_HWLOC_NAME}>)
-    set(UMF_PRIVATE_LIBRARY_DIRS ${UMF_PRIVATE_LIBRARY_DIRS}
-                                 ${LIBHWLOC_LIBRARY_DIRS})
-else()
-    set(UMF_COMMON_COMPILE_DEFINITIONS ${UMF_COMMON_COMPILE_DEFINITIONS}
-                                       "UMF_NO_HWLOC=1")
-endif()
+set(UMF_SOURCES ${UMF_SOURCES} ${HWLOC_DEPENDENT_SOURCES}
```

Comment:
this 3 lines could e merged with common sources, libs and dirs lists

Reply:
Merged sources and libs. Dirs list can't be merged with a jemalloc-dependent list.